asdf/asdf.py

```diff
@@ -28,7 +28,7 @@
 from .exceptions import AsdfDeprecationWarning
 from .extension import AsdfExtensionList, default_extensions
 
-from .tags.core import AsdfObject, Software, HistoryEntry, ExtensionMetadata
+from .tags.core import AsdfObject, Software, HistoryEntry, ExtensionMetadata, Software
```

Comment:
OK, I give up. How does importing Software twice change things? :-)

Reply:
Ha!  It's, uh, a test for code review quality.  Like Van Halen's brown M&Ms.